app/components/layers/useCapitalProjectsLayer.client.tsx

```diff
@@ -79,8 +92,10 @@ export function useCapitalProjectsLayer() {
       });
     },
     updateTriggers: {
-      getFillColor: [managingCode, capitalProjectId],
-      getPointColor: [managingCode, capitalProjectId],
+      getFillColor: [managingCode, capitalProjectId, managingAgency],
```

Comment:
Removing `managingAgency` here and on the following line doesn't seem to break anything and would cut down on how many times these functions are executed.

Reply:
Done.